build-lib/testing.js

```diff
@@ -51,14 +51,20 @@ function testsTree(pathConfig) {
   ]);
 
   const testModules = babelTranspiler(mergeTrees([testJs, linterTests]), babelConfig(`${pkg.name}/tests`, 'amd'));
+  const testsCommon = babelTranspiler(testJs, babelConfig(pkg.name, 'commonjs'));
```

Comment:
Also, if we now have a common tree and an amd tree, maybe use a similar naming convention for both, like in the other bits of build-lib? (suggestion: `testAmdTree` and `testCommonTree`)